readme.md

```diff
@@ -107,8 +107,8 @@ Below is a table showing the mapping of the LTS branches to the package versions
 
 | NPM Package | Github Branch | LTS Status | LTS Start Date | Maintenance End Date | Removed Date |
 | :-----------: | :-----------: | :--------: | :------------: | :------------------: | :----------: |
-| 1.7.x         | [lts_09_2018](https://github.com/Azure/azure-iot-sdk-node/tree/lts_09_2018)   | Deprecated | 2018-09-16     | 2019-09-16           | 2020-03-09   |
 | 1.11.x        | [lts_09_2019](https://github.com/Azure/azure-iot-sdk-node/tree/lts_09_2019)   | Active     | 2019-09-09     | 2020-09-09           | 2021-03-09   |
+| 1.17.x        | [lts_07_2020](https://github.com/Azure/azure-iot-sdk-node/tree/lts_07_2020)   | Active     | 2020-07-28     | 2021-07-28           | 2022-01-28   |
 
 <sup>1</sup> All scheduled dates are subject to change by the Azure IoT SDK team.
 
```

Comment:
a minor suggestion-- could link to the current explanation of our LTS policy:

"To learn more about our SDK's current LTS policy, please see [this page](https://azure.microsoft.com/en-us/blog/iot-sdk-lts-branch/)."

Reply:
Will do!